lib/containers/bats.pm

```diff
@@ -259,5 +259,6 @@ sub bats_post_hook {
         upload_logs($log_dir . $log);
     }
 
+    upload_logs('/proc/config.gz');
```

Comment:
One might change the filename, as `config.gz` is a bit too generic.

Reply:
This is the kernel config path which I checked in SLE & openSUSE.  Using `/boot/config-$(uname -r)` as fallback is overkill but I'll add it when we encounter this scenario.